csharp/Platform.Collections/Arrays/ArrayFiller[TElement, TReturnConstant].cs

```diff
@@ -1,29 +1,82 @@
 ﻿using System.Collections.Generic;
 using System.Runtime.CompilerServices;
 
-#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
-
 namespace Platform.Collections.Arrays
 {
+    /// <summary>
+    /// <para>Provides <see cref="T:TElement[]"/> array filler with additional methods that return a given constant of type <see cref="TReturnConstant"/>.</para>
+    /// <para>Предоставляет заполнитель массива <see cref="T:TElement[]"/> c дополнительными методами, возвращающими заданную константу типа <see cref="TReturnConstant"/>.</para> 
+    /// </summary>
+    /// <typeparam name="TElement"><para>The elements' type.</para><para>Тип элементов массива.</para></typeparam>
+    /// <typeparam name="TReturnConstant">The return constant' type.</typeparam>
     public class ArrayFiller<TElement, TReturnConstant> : ArrayFiller<TElement>
     {
         protected readonly TReturnConstant _returnConstant;
 
+        /// <summary>
+        /// <para>Initializes a new instance of the class <see cref="ArrayFiller"/> using the specified array, the offset from which filling will start and the constant returned when filling elements.</para>
```

Comment:
`the <see cref="ArrayFiller"/> class` instead of `the class <see cref="ArrayFiller"/>`